src/urls.ts

```diff
@@ -4,9 +4,18 @@
 const resolveUrlProtocol = (saleorDomain: string): string =>
   saleorDomain === "localhost:8000" ? "http" : "https";
 
+/**
+ * @deprecated use getJwksUrlFromSaleorApiUrl
+ */
 export const getJwksUrl = (saleorDomain: string): string =>
   `${resolveUrlProtocol(saleorDomain)}://${saleorDomain}/.well-known/jwks.json`;
 
+export const getJwksUrlFromSaleorApiUrl = (saleorApiUrl: string): string =>
+  `${new URL(saleorApiUrl).origin}/.well-known/jwks.json`;
```

Comment:
@przlada I assume path to jwks.json is always static

Reply:
In the future we'll not need to do this shenanigans: https://github.com/saleor/saleor/issues/10800